code/modules/spells/spell_types/pointed/mind_transfer.dm

```diff
@@ -39,6 +39,20 @@
 		if(stand.summoner)
 			victim = stand.summoner
 
+	//You should not be able to enter one of the most powerful side-antags as a fucking wizard.
+	if(istype(victim,/mob/living/simple_animal/slaughter))
+		to_chat(user, "<span class='warning'>The devilish contract doesn't include the 'mind swappable' package, please try again another lifetime.</span>")
+		user.death(FALSE) //fancy suicide
+		user.ghostize(FALSE)

```

Comment:
why do you punish players by completely ending their round if they try to do a funny? just make a message and get it over with